src/System.Runtime.Extensions/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -1,7 +1,8 @@
 Compat issues with assembly System.Runtime:
-MembersMustExist : Member 'System.IO.Path.Join(System.String, System.String)' does not exist in the implementation but it does exist in the contract.
```

Comment:
The changes in the ApiCompatBaseline.uapaot.txt files won't be necessary once your CoreRT change propagates through the system. You can undo them.